tests/test_comparison.py

```diff
@@ -68,6 +68,7 @@ def test_unequal_grains_that_differ_at_specific_points_compare_as_unequal(self,
         self.assertIn(excl, c.failing_attributes())
 
     @given(sampled_from(GRAIN_TYPES_TO_TEST).flatmap(attribute_and_pairs_of_grains_of_type_differing_only_in_one_attribute))
+    @settings(deadline=None)
```

Comment:
I think I understand why this test is quite slow (the examples are quite complex), but a comment explaining why the deadline is turned off is probably warranted at least